content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -52,6 +52,7 @@ different Kubernetes components.
 | `APIListChunking` | `false` | Alpha | 1.8 | 1.8 |
 | `APIListChunking` | `true` | Beta | 1.9 | |
 | `APIPriorityAndFairness` | `false` | Alpha | 1.17 | |
```

Comment:
the convention seems to be to have the last alpha version in the Until column - do we do this only if the alpha API is entirely removed?

Reply:
Added the "until" for alpha.